persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/SampleListMapper.xml

```diff
@@ -32,11 +32,16 @@
         INNER JOIN cancer_study ON sample_list.CANCER_STUDY_ID = cancer_study.CANCER_STUDY_ID
         INNER JOIN reference_genome ON cancer_study.REFERENCE_GENOME_ID = reference_genome.REFERENCE_GENOME_ID
         <if test="studyIds != null">
-			WHERE cancer_study.CANCER_STUDY_IDENTIFIER IN
-			<foreach item="item" collection="studyIds" open="("
-				separator="," close=")">
-				#{item}
-			</foreach>
+            <if test="studyIds.isEmpty()">
+                WHERE FALSE
+            </if>
+            <if test="!studyIds.isEmpty()">
```

Comment:
Nice! We have lots of places are require non-empty study id list or non-empty sample/patient list, maybe we can do the same thing for those mapper methods.